app/assets/scripts/components/road-table-row-properties.js

```diff
@@ -9,19 +10,48 @@ const RowProperties = ({ properties }) => (
     className="table-properties"
   >
     <dl className='table-properties-list'>
+      <LengthChart
+        platformLength={properties['length']}
+        tabularLength={properties['Road Length (VProMMS)']}
+      />
       {
-        map(properties, (prop, key) => (
-          <div
-            key={key}
-          >
-            <dt>{key}</dt>
-            <dd>{prop}</dd>
-          </div>
-        ))
+        map(properties, (prop, key) =>
+          // Since we have the above chart, no need to include length values
+          key.startsWith('Road Length') || key === 'length'
```

Comment:
we could just filter out these properties via `_.filter`, yeah?

Reply:
also, do we need a polyfill for `startsWith`?  Not sure what browsers we are targeting--apparently IE isn't supported: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

Reply:
Good thought on the compatibility side! Fortunately we have Babel tasks to compile down the advanced syntax.

Reply:
`_.filter` is for collections, rather than Objects. I think it reads relatively simply right now, so I'm going to keep it as-is, but good thought.